std-bits/table/src/main/java/org/enso/table/data/table/Table.java

```diff
@@ -43,6 +43,11 @@ public Table(Column[] columns, AggregatedProblems problems) {
       throw new IllegalArgumentException("A Table must have at least one column.");
     }
 
+    boolean uniqueColumns = Arrays.stream(columns).map(Column::getName).distinct().count() == columns.length;
```

Comment:
Can we do this as a function with a HashSet failing early?
Computing the complete counts feels inefficient.